sass/layout/_sidebar-left.scss

```diff
@@ -1094,32 +1098,31 @@
                 text-overflow: ellipsis;
             }
         }
-    }
 
-    .SidebarChannel.animating {
-        transition-duration: 0.15s, 0.15s, 0.15s, 0.15s, 0.15s; /* collapse animation speed */
-        transition-property: height, padding, color, opacity, visibility;
-        transition-timing-function: ease, ease, ease, ease-in, step-start;
+        &.expanded {
+            animation-direction: normal;
+            animation-duration: 0.4s;
```

Comment:
If you notice transition duration and animation duration are different: 0.3 and 0.4 respectively. Take a look at the fps comparison with both being dissimilar and being same.

- With dissimilar duration

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/17708702/152702333-1538f43b-e485-4d64-aff2-58d12779faaa.png">

- With same duration
<img width="1356" alt="image" src="https://user-images.githubusercontent.com/17708702/152702360-1a8c7d3b-12d5-45f9-ae2e-1481218eb7ab.png">

The one with the same duration shows few frames were dropped at the starting of the menu collapsing. This reduced in frame rate resulted in choppy transition. On the other hand the one with different duration shows no frame drop resulting in near 60 fps. Height collapsing is a layout shift operation which requires more effort than opacity since browser has to paint change in height in every step. Change in duration gives browser enough room to perform the transition.